http/src/request/template/create_guild_from_template.rs

```diff
@@ -12,7 +12,7 @@ use twilight_validate::request::{guild_name as validate_guild_name, ValidationEr
 #[derive(Serialize)]
 struct CreateGuildFromTemplateFields<'a> {
     name: &'a str,
-    icon: Option<&'a [u8]>,
+    icon: Option<&'a str>,
```

Comment:
This field is optional, see https://discord.com/developers/docs/resources/guild-template#create-guild-from-guild-template